submissions/chris-voitova/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,318 @@
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="47" />
+          <span class="hall-item__seat">47</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="48" />
+          <span class="hall-item__seat">48</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="49" />
+          <span class="hall-item__seat">49</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="50" />
+          <span class="hall-item__seat">50</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="51" />
+          <span class="hall-item__seat">51</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="52" />
+          <span class="hall-item__seat">52</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="53" />
+          <span class="hall-item__seat">53</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="54" />
+          <span class="hall-item__seat">54</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="55" />
+          <span class="hall-item__seat">55</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="56" />
+          <span class="hall-item__seat">56</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="57" />
+          <span class="hall-item__seat">57</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="58" />
+          <span class="hall-item__seat">58</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="59" />
+          <span class="hall-item__seat">59</span>
+        </label>
+        <label class="hall-item">
+          <input type="checkbox" class="hall-item__input" value="60" />
+          <span class="hall-item__seat">60</span>
+        </label>
+      </div>
+      <div class="submit-block">
+        <button type="button" class="button-submit">Done</button>
+      </div>
+      <div class="summary-info">
+        <h2 class="summary-info__title">You choosed</h2>
+        <ul class="summary-list">
+          <li class="summary-list__item"><strong>Movie:</strong> Matrix</li>
+          <li class="summary-list__item"><strong>Time:</strong> 16:00</li>
+          <li class="summary-list__item"><strong>Seats:</strong> 35, 36</li>
+        </ul>
+      </div>
```

Comment:
As for me, block with summary info after submit button is a bad idea, because the user will not read this block, especially people with screenreaders when they will press submit button and they won't even know about this block because it after submit button.
What do you think about that point?

Reply:
Hmm. I would like to display this information somewhere. Maybe I can place it before the submit button and update this information as the fields are selected